assets/manifest.json

```diff
@@ -37,6 +38,7 @@
   "permissions": [
     "https://github.com/",
     "https://gist.github.com/",
-    "https://githublinker.herokuapp.com/"
+    "https://githublinker.herokuapp.com/",
+    "*"
```

Comment:
Changing the permission like that would result in a less restricted extension. The permissions message would change from 

_Read and change your data on gist.github.com, github.com and githublinker.herokupapp.com_

 to 

_Read any change all your data on the websites you visit_

Browser extensions are really powerful and they can do a lot with your browser. The permissions are a good indicator for the user to see how risky an extension could be. Therefore every extensions developer should aim for less as possible restrictions.  

In #75 I mentioned the [optional permission api](https://developer.chrome.com/extensions/permissions) which I think is the best way to deal with additional github enterprise domains.